recipes/newrelic/apm/dotNet/windows-iis.yml

```diff
@@ -159,25 +159,46 @@ install:
 
           $uninstallIds = Find-UninstallGuids -Match "New Relic .NET Agent"
           foreach ($uninstallId in $uninstallIds) {
-            try {
-              $uninstallCommand = "msiexec /x """ + $uninstallId + """ /qn"
-              Invoke-Expression $uninstallCommand -ErrorAction SilentlyContinue | Out-Null
-              # wait here to allow uninstall command to complete since it returns without waiting
-              Start-Sleep -s 15
-            }
-            catch {}
+            Start-Process -FilePath msiexec.exe -ArgumentList "/qn","/x $uninstallId" -Wait
```

Comment:
Oh... interesting... So we just waited 15 seconds and assumed the uninstall was complete? That could have caused some problems... and guarantees a 15 second wait :(

Reply:
Will a failed return code or exception in the call to Start-Process be reported? I see the old code just swallowed any errors :(

Reply:
The old code (and new code since the call is not `-PassThru`) swallowed uninstall errors because they really aren't all that significant with the msi install step later on being able to handle upgrades (and will report errors there).